simde/arm/neon/cvt.h

```diff
@@ -266,7 +274,7 @@ simde_vcvtd_s64_f64(simde_float64 a) {
       return INT64_MIN;
     } else if (HEDLEY_UNLIKELY(a > HEDLEY_STATIC_CAST(simde_float64, INT64_MAX))) {
       return INT64_MAX;
-    } else if (simde_math_isnanf(a)) {
+    } else if (simde_math_isnan(a)) {
       return 0;
```

Comment:
According to the function definitions `int isnan(double x)` and `int isnanf(float x)`, I think it should use `isnan` since the input parameter is of type `double` (`float64`).